src/cpu/flexcpu/SimpleDataflowCPU.py

```diff
@@ -28,13 +28,22 @@
 
 
 from BaseCPU import BaseCPU
+from BranchPredictor import *
 from m5.params import *
 from m5.proxy import *
 
 class SimpleDataflowCPU(BaseCPU):
     type = 'SimpleDataflowCPU'
     cxx_header = 'cpu/flexcpu/simple_dataflow_cpu.hh'
 
+    branch_pred = Param.BranchPredictor(TournamentBP(
```

Comment:
But all our other parameters are snake case... I don't like having just one different.

Reply:
See https://github.com/darchr/gem5/blob/master/src/cpu/minor/MinorCPU.py#L276 and https://github.com/darchr/gem5/blob/master/src/cpu/o3/O3CPU.py#L159. I know those two places are wrong, but we have to follow that if we want others to use this CPU model. It will break code that's general for CPU models if we name our branch predictor something different.

Reply:
dang